pkg/telemetry/span/spans.go

```diff
@@ -21,8 +21,17 @@ const (
 
 // Collector component spans
 const (
-	CollectorStream   = "kubehound.collector.stream"
-	CollectorReadFile = "kubehound.collector.readFile"
+	CollectorStream          = "kubehound.collector.stream"
+	CollectorDump            = "kubehound.collector.dump"
+	CollectorReadFile        = "kubehound.collector.readFile"
+	CollectorS3Push          = "kubehound.collector.s3_push"
+	CollectorS3Download      = "kubehound.collector.s3_download"
+	CollectorFileWriterWrite = "kubehound.collector.file_writer.write"
```

Comment:
I did not go in this direction, because I wanted to capture the network latency with the s3 push and download which does not exist with the file writer (only local).